sktime/forecasting/model_selection/_tune.py

```diff
@@ -526,6 +541,21 @@ def __init__(
         self.n_iter = n_iter
         self.random_state = random_state
 
+    def get_fitted_params(self):
```

Comment:
Nice! Both estimators have this though, we should move this up to the common parent class, `BaseGridSearch`.